src/buildercore/checks.py

```diff
@@ -9,7 +9,9 @@ class AccessProblem(RuntimeError):
     pass
 
 class StackAlreadyExistsProblem(RuntimeError):
-    pass
+    def __init__(self, message, stackname):
+        RuntimeError.__init__(self, message)
```

Comment:
ah, another python3-ism: https://www.journaldev.com/15911/python-super

Reply:
can `super` be used in both Python versions?

Reply:
looks like it. Python3:

```python3
Python 3.7.0 (default, Sep 15 2018, 19:13:07) 
[GCC 8.2.1 20180831] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> class Foo:
...     pass
... 
>>> class Bar(Foo):
...     def __init__(self):
...         super(Bar, self).__init__()
... 
>>> baz = Bar()
```

python2:
```python
Python 2.7.15 (default, Jun 27 2018, 13:05:28) 
[GCC 8.1.1 20180531] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> class Foo(object):
...     pass
... 
>>> class Bar(Foo):
...     def __init__(self):
...         super(Bar, self).__init__()
... 
>>> baz = Bar()
```

Reply:
with python2 you have to use 'new style' classes though, like the above